### PR TITLE
Add Customer websiteId

### PR DESCRIPTION
**Pull request by colinbird**

### CODE REVIEW

File: src/Entities/Customer.php [hunk 1]
```diff
@@ -350,8 +350,26 @@ public function getInvoiceCount()
...
     {
         return $this->setAttribute('paymentToken', $value);
     }
+
+    /**
+     * @param string $value
+     *
+     * @return Customer
+     */
+    public function setCustomerId(string $value)
```
Comment (slavcodev) [inline src/Entities/Customer.php]:
```suggestion
    public function setCustomerId(?string $value)
```

File: src/Entities/Customer.php [hunk 2]
```diff
@@ -350,8 +350,26 @@ public function getInvoiceCount()
...
+    {
+        return $this->setAttribute('websiteId', $value);
+    }
+
+    /**
+     * @return string
+     */
+    public function getWebsiteId()
```
Comment (slavcodev) [inline src/Entities/Customer.php]:
```suggestion
    public function getWebsiteId(): ?string
```

File: src/Entities/Customer.php [hunk 3]
```diff
@@ -350,8 +350,26 @@ public function getInvoiceCount()
      *
      * @return $this
      */
-    public function setPaymentToken($value)
+    public function setPaymentToken(string $value)
```
Comment (slavcodev) [inline src/Entities/Customer.php]:
```suggestion
    public function setPaymentToken(?string $value)
```

**Review by slavcodev — COMMENTED** [review]
Also you need to update CHANGELOG

**Comment by colinbird** [comment]
@slavcodev should I make a new version?

File: CHANGELOG.md [hunk 4]
```diff
@@ -16,6 +16,10 @@ Security - in case of vulnerabilities.
 
 ## [Unreleased]
 
+## [2.4.1] 2019-07-18
```
Comment (slavcodev) [inline CHANGELOG.md]: Not necessarily, we do not release this library for each change. According symver you can add change log under `[Unreleased]`.

Comment (ashkarpetin) [reply CHANGELOG.md]: I agree, we can add change log under `Unreleased`

**Review by dara123 — APPROVED** [review]

File: CHANGELOG.md [hunk 5]
```diff
@@ -16,6 +16,9 @@ Security - in case of vulnerabilities.
 
 ## [Unreleased]
 
+### Added
+- [x] Added websiteId to Customer.
```
Comment (ashkarpetin) [inline CHANGELOG.md]:
```suggestion
- [x] Added new property to `Customer`: `websiteId`
```

**Review by ashkarpetin — APPROVED** [review]

**Pull request merged**